packages/shared/src/hooks/useSquadInvitation.ts

```diff
@@ -25,14 +25,7 @@ export const useSquadInvitation = ({
   const { completeAction } = useActions();
 
   const invitation = useMemo(() => {
-    const permalink = squad?.permalink;
-    const token = squad?.currentMember?.referralToken;
-
-    if (!permalink || !token) {
-      return undefined;
-    }
-
-    return `${permalink}/${token}`;
+    return squad.referralUrl;
```

Comment:
I think in this case we don't need the `useMemo` anymore.

Reply:
Yep, agreed.